### PR TITLE
Jfb 2

### PR DESCRIPTION
**Pull request by RamazArtmeladze**

### CODE REVIEW

**Comment by RamazArtmeladze** [comment]
test comment

**Comment by DarkBeer0** [comment]
there's no conflict here, congratulations!

File: src/TicketService.java [hunk 1]
```diff
@@ -17,4 +19,15 @@ public TicketService() {
...
+    public List<Ticket> getTicketsByStadiumSector(char stadiumSector) {
+        return tickets.values().stream()
+                .filter(ticket -> ticket.getStadiumSector() == stadiumSector)
+                .collect(Collectors.toList());
+    }
+
+    public static void main(String[] args) {
+        TicketService ticketService = new TicketService();
+        System.out.println("Tickets in sector A: " + ticketService.getTicketsByStadiumSector('A'));
+    }
```
Comment (RamazArtmeladze) [inline src/TicketService.java:32]: test comment

File: src/TicketService.java [hunk 2]
```diff
@@ -1,16 +1,33 @@
...
+        return tickets.get(id);
+    }
 
-        Ticket limitedTicket = new Ticket("SmallHall", 456, Instant.now().getEpochSecond());
-        System.out.println("Limited Ticket: " + limitedTicket);
-        //test
+    public List<Ticket> getTicketsByStadiumSector(char stadiumSector) {
+        return tickets.values().stream()
+                .filter(ticket -> ticket.getStadiumSector() == stadiumSector)
+                .collect(Collectors.toList());
```
Comment (deolexx) [inline src/TicketService.java:26]: .collect(Collectors.toList()); could be replaced with just toList();

File: src/TicketService.java [hunk 3]
```diff
@@ -1,16 +1,33 @@
...
+    }
+
+    public static void main(String[] args) {
+        TicketService ticketService = new TicketService();
+        System.out.println("Tickets in sector A: " + ticketService.getTicketsByStadiumSector('A'));
```
Comment (deolexx) [inline src/TicketService.java:31]: For now it is ok but in general using System.out.println for logging in production code is not recommended

**Review by deolexx — APPROVED** [review]

**Pull request merged**